packages/datadog-plugin-fastify/test/index.spec.js

```diff
@@ -25,6 +26,8 @@ describe('Plugin', () => {
 
       withExports('fastify', version, ['default', 'fastify'], '>=3', getExport => {
         describe('without configuration', () => {
+          if ((NODE_MAJOR <= 18) && semver.satisfies(specificVersion, '>=5')) return
```

Comment:
Why these extra params?
```suggestion
          if (NODE_MAJOR <= 18 && semver.satisfies(specificVersion, '>=5')) return
```

Reply:
I just copy pasted from somewhere else without checking 😆 , good catch!